docs/conf.py

```diff
@@ -46,20 +46,25 @@
     'sphinx.ext.viewcode',
     'sphinx.ext.napoleon',
     'recommonmark',
+    'sphinxawesome_theme',
 ]
 
 templates_path = ['_templates']
 exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
 
 # -- Options for HTML output -------------------------------------------------
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output
+html_permalinks_icon = "<span>^</span>"
+html_theme = "sphinxawesome_theme"
 
-html_theme = 'sphinx_book_theme'
-
+# Update theme options for sphinx_awesome_theme
 html_theme_options = {
-    "logo": {
-        "text": f"CAMEL {release}",
-        "image_light": "https://raw.githubusercontent.com/camel-ai/camel/master/misc/primary_logo.png",
-        "image_dark": "https://raw.githubusercontent.com/camel-ai/camel/master/misc/primary_logo.png",
+    "navbar_start": ["navbar-logo"],
+    "navbar_center": ["navbar-nav"],
+    "navbar_end": ["navbar-icon-links"],
+    "navbar_persistent": ["search-button"],
+    "logo_light": "https://raw.githubusercontent.com/camel-ai/camel/master/misc/logo_light.png",
```

Comment:
did you see this works locally with that url changed temporarily to one that resolves? I think this should be fixed before merging, otherwise I expect the logo to show as a broken image in the docs

Reply:
I have the same issue with Caelumn, the image is broken